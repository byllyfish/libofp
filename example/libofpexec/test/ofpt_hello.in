{id: 1, method: ofp.send, params: {datapath_id: '1234567890abcdef', auxiliary_id: 1, type: OFPT_HELLO, version: 4, xid: 0x01020304, msg: {versions: [1, 2, 3, 4]}}}
{id: 2, method: ofp.send, params: {type: OFPT_HELLO, xid: 0x01020304, version: 1}}
{id: 3, method: ofp.send, params: {type: OFPT_HELLO}}
{id: 4, method: ofp.send, params: {type: OFPT_HELLO, msg: {versions: [1]}}}
{id: 5, method: ofp.send, params: {type: OFPT_HELLO, version: 1, msg: {versions: [1, 2]}}}
{id: 6, method: ofp.send, params: {type: OFPT_HELLO, version: 2, msg: {versions: [1, 4]}}}
{id: 7, method: ofp.send, params: {type: OFPT_HELLO, version: 3, msg: {versions: [1, 3]}}}
{id: 8, method: ofp.send, params: {type: OFPT_HELLO, version: 5, msg: {versions: [1,3]}}}
{id: 9, method: ofp.send, params: {type: OFPT_HELLO, version: 5, msg: {versions: [0]}}}
{id: 10, method: ofp.send, params: {type: OFPT_HELLO}}
