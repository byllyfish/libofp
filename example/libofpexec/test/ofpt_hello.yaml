---
type:            OFPT_HELLO
xid:             0x01020304
version:         0x04
msg:             
  versions:        [ 1, 2, 3, 4 ]
...
---
type:            OFPT_HELLO
xid:             0x01020304
version:         0x01
msg:             
  versions:        [  ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x04
msg:             
  versions:        [ 1, 2, 3, 4 ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x01
msg:             
  versions:        [  ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x01
msg:             
  versions:        [  ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x02
msg:             
  versions:        [  ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x03
msg:             
  versions:        [  ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x05
msg:             
  versions:        [ 1, 3 ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x05
msg:             
  versions:        [ 5 ]
...
---
type:            OFPT_HELLO
xid:             0x00000000
version:         0x04
msg:             
  versions:        [ 1, 2, 3, 4 ]
...
