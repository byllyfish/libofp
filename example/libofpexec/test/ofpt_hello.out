---
event:           LIBOFP_LOOPBACK
params:          
  data:            0400001001020304000100080000001E
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0100000801020304
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0400001000000000000100080000001E
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0100000800000000
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0200000800000000
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            04000010000000000001000800000012
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0300000800000000
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0300000800000000
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0400001000000000000100080000001E
...
---
event:           LIBOFP_LOOPBACK
params:          
  data:            0400001000000000000100080000001E
...
